includes/ppc-tabs.php

```diff
@@ -45,10 +45,14 @@
 		echo 'nav-tab-active ppc-active-tab';
 	}
 	?>
-	"><?php esc_attr_e( 'General Settings', 'pre-publish-checklist' ); ?></a>
-
-
-		<a href="?page=ppc&tab=ppc-checklist" class="nav-tab tb 
+	">
+	<?php esc_attr_e( 'General Settings', 'pre-publish-checklist' ); ?></a>
+		<?php
+		$ppc_post_types       = get_option( 'ppc_post_types_to_display' );
+		$ppc_type             = isset( $ppc_post_types[0] ) ? $ppc_post_types[0] : '';
+		$ppc_active_checkpost = ! empty( $ppc_post_types ) ? $ppc_type : 'post';
+		?>
+		<a href="?page=ppc&tab=ppc-checklist&type=<?php echo esc_attr( $ppc_active_checkpost ); ?>" class="nav-tab tb 
```

Comment:
Using add_query_arg for anchor link would be better here. [https://developer.wordpress.org/reference/functions/add_query_arg/](url)

Reply:
Used  add_query_arg